src/sentry/static/sentry/app/views/organizationSettings.jsx

```diff
@@ -82,8 +83,12 @@ const OrganizationSettingsForm = React.createClass({
         method: 'PUT',
         data: {
           ...formData,
-          safeFields: formData.safeFields.split('\n'),
-          sensitiveFields: formData.sensitiveFields.split('\n'),
+          safeFields: (
+            trim(formData.safeFields).split('\n').filter(f => f !== '')
```

Comment:
I think more needs to be done here (this is using the regex from `utils.split`):

```javascript
'   derp      derp derp\n    derp derp\n      \ntest'
  .replace(/^\s+|\s+$/g,'').split('\n')
  .filter(f => f !== '');

// => ["derp      derp derp", "    derp derp", "      ", "test"]
```